apps/web/test12.html

```diff
@@ -116,13 +116,14 @@
       const page = pdfDoc.addPage(PageSizes.Letter);
 
       // quadratic bezier example
-      page.drawSvgPath('M200,300 Q400,50 600,300 T1000,300', {
-        x: inchToPt(-1),
-        y: inchToPt(11),
+      page.drawSvgPath('M0,0 q100,125 100,0 T200,0', {
```

Comment:
Was there a reason we needed to change this SVG path? I believe the original path came from https://svgwg.org/svg2-draft/paths.html, so it would be nice to keep that one unless we really need to change it.

Reply:
I just wanted to try how the change of the "movement" command (`M200,300` to `M0,0`) affects the origin (pivot) of the rotation. I've reverted this change now, as well. Let me know if there's anything else!